crates/rust-analyzer/src/lsp_ext.rs

```diff
@@ -98,6 +98,22 @@ pub struct JoinLinesParams {
     pub ranges: Vec<Range>,
 }
 
+pub enum ResolveCodeActionRequest {}
+
+impl Request for ResolveCodeActionRequest {
```

Comment:
Could you also add docs for this to https://github.com/rust-analyzer/rust-analyzer/blob/21132a7a748a70351f5d1ae6d8c51a3a0065013e/docs/dev/lsp-extensions.md?